spec/intl.html

```diff
@@ -2139,13 +2182,22 @@ <h1>Temporal.Calendar.prototype.mergeFields ( _fields_, _additionalFields_ )</h1
             1. Perform ? RequireInternalSlot(_calendar_, [[InitializedTemporalCalendar]]).
```

Comment:
In a post-#2482 world, does this Calendar method get called when a string id was used for the calendar? Or is there some AO that's called instead?

I'm asking because looking at the spec, Temporal will never call `mergeFields` with either `era` or `eraYear` present in `fields`. So "the returned List should contain *"era"* and *"eraYear"*" is a no-op unless userland code is calling `mergeFields`. Is this correct?

If so, then should there be a simpler version of this implementation that's used for non-observable merge-fields operations?  Or does that already exist?